src/examples/holy-grail.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx jsx */
+import { jsx, Box, Flex } from "../code";
+
+export default () => (
+  <Flex sx={{ flexDirection: "column", minHeight: "100vh" }}>
+    <Box>Header</Box>
+    <Flex sx={{ flex: 1, flexDirection: ["column", "row"] }}>
+      <Box as="main" sx={{ flex: 1 }}>
```

Comment:
These are being rendered with the special `jsx` create element function to be able to return code strings in different syntaxes, so `as` will not work. I'd like to add a section to the site to show how to add HTML sectioning elements, but don't want to make assumptions about content semantics in the layout examples... but that new section is sort of out of scope for this PR

Reply:
Makes sense. Adjusted column width and copy in 17566ccd2b336885ca07a5483bd092c4fe844884.